cookbook/src/intro.md

```diff
@@ -1,36 +1,25 @@
-# Rust Bitcoin
+# Welcome!
```

Comment:
Titles should describe what the page is about - in this case `# Introduction to Rust Bitcoin` seems like a good option.

Reply:
I agree with you in general but in this instance we have "Rust Bitcoin Cookbook" show right above in the "title bar" (not sure of term) so having "Introduction to Rust Bitcoin" and "introduction" in the sidebar seems a bit redundant and, totally subjectively, a bit dry. I don't really mind though. Thoughts?

Reply:
Maybe just "Introduction" then. The thing on the left should match the title.

Reply:
Works for me!